server/qrops.go

```diff
@@ -0,0 +1,281 @@
+package server
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"image"
+	"net/url"
+	"strconv"
+
+	"github.com/yeqown/go-qrcode/v2"
+	"github.com/yeqown/go-qrcode/writer/standard"
+	"go.uber.org/zap"
+
+	"github.com/status-im/status-go/images"
+	"github.com/status-im/status-go/multiaccounts"
+)
+
+type WriterCloserByteBuffer struct {
+	*bytes.Buffer
+}
+
+func (wc WriterCloserByteBuffer) Close() error {
+	return nil
+}
+
+func NewWriterCloserByteBuffer() *WriterCloserByteBuffer {
+	return &WriterCloserByteBuffer{bytes.NewBuffer([]byte{})}
+}
+
+type QRConfig struct {
+	DecodedQRURL    string
+	WithLogo        bool
+	CorrectionLevel qrcode.EncodeOption
+	KeyUID          string
+	ImageName       string
+	Size            int
+	Params          url.Values
+	Logger          *zap.Logger
```

Comment:
It is very rare that you would need to expose a logger to any external code. I'd suggest making this unexported.

```suggestion
	logger          *zap.Logger
```

Reply:
got rid of this in : https://github.com/status-im/status-go/pull/3172/commits/b55243dbf691338c82ca6c09e00042f7c8decfe1